packages/table/src/common/utils.ts

```diff
@@ -42,7 +44,7 @@ export const Utils = {
      * element's original className and any other classes passed in with variadic
      * arguments matching the `classNames` api.
      */
-    assignClasses<P extends IProps>(elem: React.ReactElement<P>, ...extendedClasses: ClassValue[]) {
+    assignClasses<P extends IProps>(elem: React.ReactElement<P>, ...extendedClasses: ClassValue[]): React.ReactNode {
```

Comment:
Why all the explicit return signatures?

Reply:
I find it good hygiene for public-ish APIs. Not entirely relevant to the change at hand.